packages/sdk/src/mb-impl/token.ts

```diff
@@ -0,0 +1,214 @@
+// Mintbase token contract JS implementation
+
+import { TransactionArgs } from '../calls';
+import {
+  DEFAULT_MB_LOGO,
+  METHOD_NAMES,
+  MB_MAINNET_TOKEN_FACTORY,
+  MB_TESTNET_TOKEN_FACTORY,
+  Network,
+} from './constants';
+import {
+  TransferArgs,
+  BurnArgs,
+  SingleTokenArgs,
+  DeployTokenContractArgs,
+  AccountId,
+  TransferTokenContractOwnership,
+  MintArgs,
+  AddRemoveMinterArgs,
+  BatchChangeMinters,
+  RevokeAccountArgs,
+} from './token.types';
+
+// TODO: how to deal with this differently?
```

Comment:
This type of method only become necessary in cases where the network can be switched at runtime, so is there a way to avoid this? e.g. require a refresh for browser network switches and do something like:

```
const NEAR_NETWORK = process.env.NEAR_NETWORK || global.NEAR_NETWORK || localStorage?.get('NEAR_NETWORK')
 ```
 
If we can prevent this, network will be set at environment/browser level vs switched at runtime I'm okay with setting constants based on that e.g. 
```
const TOKEN_CONTRACT_FACTORY = NEAR_NETWORK === 'mainnet' 
    ?  TOKEN_CONTRACT_FACTORY_MAINNET
    : TOKEN_CONRACT_FATORY_TESTNET
```

We can then just omit this super basic logic from test coverage (constants are omitted)